resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java

```diff
@@ -32,15 +44,65 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String fname = ServletHelpers.getParameter(request, "fname", "");
     String lname = ServletHelpers.getParameter(request, "lname", "");
     String email = ServletHelpers.getParameter(request, "email", "");
-    reviewee = new Reviewee(fname, lname, email);
+    String resumeURL = getUploadedFileUrl(request, "resume");
+    reviewee = new Reviewee(fname, lname, email, resumeURL);
+
     Entity revieweeEntity = new Entity("Reviewee");
     revieweeEntity.setProperty("first-name", fname);
     revieweeEntity.setProperty("last-name", lname);
     revieweeEntity.setProperty("email", email);
+    revieweeEntity.setProperty("resumeURL", resumeURL);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(revieweeEntity);
 
     response.sendRedirect("resume-review.html");
   }
+
+  /**
+   * @return the request parameter, or the default value if the parameter was not specified by the
+   *     client
+   */
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
+    String value = request.getParameter(name);
+    return (value == null) ? defaultValue : value;
+  }
```

Comment:
You can delete all of this, because Shreya put this helper function in the ServletHelpers static class. This file is already using the `getParameter` definition in ServletHelpers.

Reply:
Gotcha!